model/reader.go

```diff
@@ -509,7 +515,10 @@ func (r *PdfReader) GetOutlines() (*Outline, error) {
 
 // RepairAcroForm attempts to rebuild the AcroForm fields using the widget
 // annotations present in the document pages.
-func (r *PdfReader) RepairAcroForm() error {
+// NOTE: the opts parameter is not used currently as the repair process does
```

Comment:
Can also just say: Pass nil for the options to use default options. The options parameter is declared for enabling adding future options but passing nil will always use the default options.
Should always be safe to pass the nil value for default parameters.